spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -417,33 +427,6 @@ public static void importSparkTable(
     }
   }
 
-  /**
-   * Import files from an existing Spark table to an Iceberg table.
-   *
-   * <p>The import uses the Spark session to get table metadata. It assumes no operation is going on
-   * the original and target table and thus is not thread-safe.
-   *
-   * @param spark a Spark session
-   * @param sourceTableIdent an identifier of the source Spark table
-   * @param targetTable an Iceberg table where to import the data
-   * @param stagingDir a staging directory to store temporary manifest files
-   * @param checkDuplicateFiles if true, throw exception if import results in a duplicate data file
-   */
-  public static void importSparkTable(
-      SparkSession spark,
```

Comment:
Important: this is a public utility, we want to maintain compatibility and not remove it. If there's a reason we want to deprecate it, it should go down a deprecation path. But I don't see why we would want to deprecate this